generator/docs/SetRestrictionAndGeneration.md

```diff
@@ -0,0 +1,114 @@
+# Set restriction process
+
+The data generator profile describes to the generator how to reduce the original set of data to a permitted set of data per field. In most cases the generator starts off using the universal set as the source of data. This represents all values for all types (temporal, string, numeric) without restriction. It also allows for no value to be emitted - the empty set (&#8709;) commonly expressed as `null`).
+
+The generator will only (effectively) use a different original set of data - i.e. not use the universal set - if one of the following constraints are used:
```

Comment:
This line technically isn't true since using ofType also reduces the universal set

Reply:
This is explained in line 15, where we describe how the `ofType` operator works. 
The ofType operator filters, where as equalTo/inSet/null replace the original set of data